parent/pom.xml

```
@@ -29,17 +29,17 @@
         <java.minimum.version>1.8</java.minimum.version>
 
         <!-- core -->
-        <commons-lang3.version>3.4</commons-lang3.version>
+        <commons-lang3.version>3.9</commons-lang3.version>
         <javax-inject.version>1</javax-inject.version>
 
         <!-- module dependencies -->
-        <jackson.version>2.9.4</jackson.version>
-        <guice.version>4.1.0</guice.version>
-        <argparse4j.version>0.4.4</argparse4j.version>
+        <jackson.version>2.10.2</jackson.version>
+        <guice.version>4.2.2</guice.version>
```

Guice version 4.2.2 already supports java11, but we still use java8, which can be upgraded later.
```suggestion
        <guice.version>4.2.0</guice.version>
```